mlir/utils/jenkins/Jenkinsfile

```diff
@@ -848,9 +863,13 @@ pipeline {
                                 sh """python3 ./bin/perfRunner.py --op=gemm --batch_all \
 					--configs_file=${WORKSPACE}/mlir/utils/performance/bert-configs \
 					--tuning_db=${WORKSPACE}/mlir_tuning_${CHIP}.tsv"""
-                                sh """python3 ./bin/perfRunner.py --op=gemm --batch_all \
-                    --configs_file=${WORKSPACE}/mlir/utils/performance/bert-configs \
-                    --tuning_db=${WORKSPACE}/mlir_tuning_${CHIP}.tsv --external-gemm-library CK"""
+                            script{
+                                if (CKBuilt == true) {
```

Comment:
This does not fit well with the stage description : "Test MLIR vs MIOpen", either we have to change the stage name or move this to a different stage such as Test MLIR vs CK.

nit : in that way, we could even use when clause to guard the stage as well..